src/type-mappers/object.ts

```diff
@@ -12,13 +12,43 @@ import {
 import { SwaggerType } from "../swagger/Swagger";
 import { Swagger } from "../swagger/Swagger";
 import { convertType } from "../typescript";
+import { makeAnyTypeSpec } from "./any";
 
 export interface ObjectTypeSpec extends TypeSpec {
   readonly tsType: "object";
   readonly isAtomic: false;
   readonly isObject: true;
   readonly requiredPropertyNames: ReadonlyArray<string>;
   readonly properties: ReadonlyArray<TypeSpec>;
+  readonly hasAdditionalProperties: boolean;
+  readonly additionalPropertiesType: TypeSpec | undefined;
+}
+
+// see: https://support.reprezen.com/support/solutions/articles/6000162892-support-for-additionalproperties-in-swagger-2-0-schemas
+export function extractAdditionalPropertiesType(
+  swaggerType: SwaggerType,
+  swagger: Swagger
+): TypeSpec | undefined {
+  if (swaggerType.type !== "object") {
+    return undefined;
+  }
+  if (swaggerType.additionalProperties === false) {
+    return undefined;
+  }
+  if (
+    swaggerType.additionalProperties === undefined ||
+    swaggerType.additionalProperties === true
```

Comment:
If `additionalProperties` is `undefined`, wouldn't it make more sense to default to `false`?